src/chains/migrations/0032_feature.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 3.2.9 on 2021-11-03 14:29
+from django.apps.registry import Apps
+from django.db import migrations, models
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations import RunPython
+
+from ..models import Feature
+
+DEFAULT_FEATURES = [
+    Feature(name="CONTRACT_INTERACTION"),
+    Feature(name="DOMAIN_LOOKUP"),
+    Feature(name="ERC721"),
+    Feature(name="ERC1155"),
+    Feature(name="SAFE_APPS"),
+    Feature(name="SPENDING_LIMIT"),
```

Comment:
A couple suggestions, the same as for Wallets:
* `name` sounds a bit like something used for display. Maybe call it `key` or `feature_id`
* It might make sense to add an internal incremental id as the primary key instead of the name.

Reply:
Thanks for the suggestions. Made the changes here: 7ed430d